src/pages/[lang]/404.astro

```diff
@@ -1,11 +1,10 @@
 ---
-import languages from '../../i18n/languages';
+import languages from '~/i18n/languages';
```

Comment:
Amazing, thanks @nikhilhenry!

I'm just going to revert this switch to using the alias for consistency with the rest of the codebase, but this is great!


```suggestion
import languages from '../../i18n/languages';
```